stories/personCard.stories.js

```diff
@@ -8,16 +8,16 @@
 import { html } from 'lit-element';
 import { withA11y } from '@storybook/addon-a11y';
 import { withKnobs } from '@storybook/addon-knobs';
+import { withWebComponentsKnobs } from 'storybook-addon-web-components-knobs';
 import { withSignIn } from '../.storybook/signInAddon';
+import { withCodeEditor } from '../.storybook/codeEditorAddon/codeAddon';
 import '../dist/es6/components/mgt-person-card/mgt-person-card';
 import '../dist/es6/components/mgt-person/mgt-person';
-import '../dist/es6/mock/mgt-mock-provider';
-import '../dist/es6/mock/MockProvider';
 
 export default {
   title: 'mgt-person-card',
-  component: 'mgt-person-card',
-  decorators: [withA11y, withKnobs, withSignIn],
+  component: 'mgt-person',
```

Comment:
This was kind of a bug before. Because the person card is generated based on person data, on load, it would throw an error because SB is looking at the dom to add component knobs. By changing this to mgt-person, properties in Knobs are no longer accurate. Because it will in turn show mgt-person properties.